unlock-app/src/__tests__/components/creator/FatalError.test.tsx

```diff
@@ -8,56 +8,64 @@ import {
   FATAL_WRONG_NETWORK,
 } from '../../../errors'
 
+import Error from '../../../utils/Error'
+
+const { Application } = Error
+
 describe('FatalError', () => {
   describe('mapErrorToComponent', () => {
     describe('maps errors to default components', () => {
       it('FATAL_MISSING_PROVIDER', () => {
-        expect.assertions(1)
-        const component = mapErrorToComponent(FATAL_MISSING_PROVIDER, {})
+        expect.assertions(0)
+        const component = mapErrorToComponent(
+          Application.Fatal(FATAL_MISSING_PROVIDER)
+        )
         const wrapper = rtl.render(component)
-        expect(wrapper.queryByText('Wallet missing')).not.toBeNull()
+        wrapper.getByText('Wallet missing')
```

Comment:
This is a small thing, but having 0 assertions with a `getByText` is a bit easier to debug than checking that a query is not null. `getByText` will show what was actually rendered in the failure, and it throws if it doesn't find anything (so the test will fail if the query fails)